mpas_analysis/ocean/ohc_timeseries.py

```diff
@@ -83,7 +114,7 @@ def ohc_timeseries(config, streamMap=None, variableMap=None):
     cp = namelist.getfloat('config_specific_heat_sea_water')
     # [kg/m3]
     rho = namelist.getfloat('config_density0')
-    factor = 1e-22*rho*cp
+    factor = 1e-22
```

Comment:
Why is this a magic number and why is the `rho*cp` scaling removed?  This is somewhat confusing.  If we don't need `rho` and `cp`, shouldn't they be removed above?